testcase/20.usersettings/src/main/java/UserSettings.java

```diff
@@ -0,0 +1,13 @@
+package main.java;
```

Comment:
why package main.java?  see most java project uses src/main/java as the root folder for storing java files:https://spring.io/guides/gs/gradle/ \nCreate the directory structure

In a project directory of your choosing, create the following subdirectory structure; for example, with mkdir -p src/main/java/hello on *nix systems:

└── src
    └── main
        └── java
            └── hello
Within the src/main/java/hello directory, you can create any Java classes you want. For simplicity’s sake and for consistency with the rest of this guide, Spring recommends that you create two classes: HelloWorld.java and Greeter.java.

src/main/java/hello/HelloWorld.java

<pre>
package hello;

public class HelloWorld {
  public static void main(String[] args) {
    Greeter greeter = new Greeter();
    System.out.println(greeter.sayHello());
  }
}
</pre>